pkg/fleet-manager/clusters.go

```diff
@@ -54,6 +54,12 @@ const (
 )
 
 func (f *FleetManager) reconcileClusters(ctx context.Context, fleet *fleetapi.Fleet) (ctrl.Result, error) {
+	controlplane := fleet.Annotations[fleetapi.ControlplaneAnnotation]
+	controlplaneSpecified := true
+	if len(controlplane) == 0 {
+		controlplaneSpecified = false
+	}
```

Comment:
can simplified: 

controlplaneSpecified := fleet.Annotations[fleetapi.ControlplaneAnnotation] != ""

Reply:
more the judge to `reconcile`, then we can resue it to do reconcileControlplane